<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>122</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="1">
     <widget class="QRadioButton" name="radioButton_1seg">
      <property name="text">
       <string>1 seg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton_15seg">
      <property name="text">
       <string>15 seg</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButton_1min">
      <property name="text">
       <string>1 min</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="radioButton_3min">
      <property name="text">
       <string>3 min</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QRadioButton" name="radioButton_10min">
      <property name="text">
       <string>10 min</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Frecuencia de Medicion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>194</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_Ver_Historial">
      <property name="text">
       <string>Ver Historial</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Historial de parametros</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Aceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
